CTCommons/MSTranscription/MSTranscriptionService.cs

```diff
@@ -18,18 +18,25 @@ public class MSTranscriptionService
         readonly ILogger _logger;
```

Comment:
Seems like you have branched of your earlier issue, you'll have to branch off of Staging, instead of your previous branch, because these changes don't belong in this PR. Revise this and ping me